Makefile

```diff
@@ -1,6 +1,19 @@
+# Use directory-local cargo root to install version-specific executables into.
+export CARGO_HOME = $(shell pwd)/.cargo
+
 # the series of builds, tests and checks that runs for pull requests. requires docker.
 .PHONY: ci
-ci: build integ
+ci: check-licenses build integ
+
+# installs cargo-deny
+.PHONY: cargo-deny
+cargo-deny:
+	cargo install --version 0.6.2 cargo-deny --no-default-features
+
+# checks each crate, and evaluates licenses. requires cargo-deny.
+.PHONY: check-licenses
+check-licenses: cargo-deny
+	cargo deny check --disable-fetch licenses
```

Comment:
This seemed weird so I looked at it and it says:

```
    -d, --disable-fetch
            Disable fetching of the advisory database

            When running the `advisories` check, the configured advisory
            database will be fetched and opened. If this flag is passed, the
            database won't be fetched, but an error will occur if it doesn't
            already exist locally.
```

This is not applicable since we are running `licenses` and not `advisories`. So this flag can be removed:

```suggestion
	cargo deny check licenses
```

Reply:
From what I can tell nothing changes if this is removed, so it is _most likely_ safe to do so. That said, it's on the TODO list to take a look at the newer versions of `cargo-deny` and it might make more sense to re-evaluate things like `--disable-fetch` and `--no-default-features` then.